app/src/main/java/app/klimatic/data/pref/AppSharedPreferencesImpl.kt

```diff
@@ -0,0 +1,30 @@
+package app.klimatic.data.pref
+
+import android.content.Context
+import android.content.SharedPreferences
+
+class AppSharedPreferencesImpl(
+    name: String,
```

Comment:
should be `private val name: String,  private val context: Context`

Reply:
Not needed.